lib/compiler/scjson-to-module.js

```diff
@@ -35,7 +27,7 @@ function generateFnDeclaration(fnName,fnBody,action){
     if(printTrace) console.log('generateFnDeclaration',fnName,fnBody);
 
     return 'function ' + fnName + FN_ARGS + '{\n' +
-        fnBody.split('\n').map(function(line){return '    ' + line;}).join('\n') + '\n' +   //do some lightweight formatting
+        fnBody + '\n' +
```

Comment:
the string scion is generating for an FSM of any substance is massive. many of these changes are an attempt to trim the fat.